addons/mrp/views/stock_move_views.xml

```diff
@@ -52,12 +52,7 @@
             <field name="inherit_id" ref="stock.view_stock_move_line_operation_tree" />
             <field name="arch" type="xml">
                 <xpath expr="//field[@name='lot_id']" position="attributes">
-		            <attribute name="context">{
-                        'active_mo_id': context.get('active_mo_id'),
-                        'active_picking_id': picking_id,
-                        'default_product_id': parent.product_id,
-                        }
-                    </attribute>
+		            <attribute name="context" operation="update">{'active_mo_id': context.get('active_mo_id')}</attribute>
```

Comment:
```suggestion
		        <attribute name="context" operation="update">{'active_mo_id': context.get('active_mo_id')}</attribute>
```
good align

Reply:
hey @MiquelRForgeFlow, you're too fast 😉 this isn't ready yet
I don't see the alignment issue here, though 🤔

Reply:
Sorry if I disturbed you before you finished the PR.

Compare:
![Selection_4129](https://github.com/user-attachments/assets/04dfd2bb-24e0-499e-86bf-b6d9a3ad8205)
![Selection_4130](https://github.com/user-attachments/assets/0ed75ade-57c3-42ff-8ddc-144d0b39a850)

Reply:
Oh, there was a TAB hiding in there. Didn't see it because with my settings they have the same width.
Thanks!!